docs/Contributing.asciidoc

```diff
@@ -733,6 +733,16 @@ Script `.circleci/build_dependencies.sh` can be also modified when major
 changes are performed, e.g. different OS version or packages from forked OBS
 project, etc.
 
+Example commands to manually fix ci-packages.txt:
+
+```
+# must be executed from root source folder
+docker run --rm -v$(pwd):/opt/testing_area registry.opensuse.org/devel/openqa/ci/containers/base:latest .circleci/build_dependencies.sh
```

Comment:
```suggestion
docker run --rm -v"$PWD:/opt/testing_area registry.opensuse.org/devel/openqa/ci/containers/base:latest" .circleci/build_dependencies.sh
```

Reply:
When running this command I get:

```
Problem: nothing provides perl(File::Map) needed by openQA-devel-4.6.1613477586.fa28c153e-lp152.3674.1.noarch
 Solution 1: do not install openQA-devel-4.6.1613477586.fa28c153e-lp152.3674.1.noarch
 Solution 2: break openQA-devel-4.6.1613477586.fa28c153e-lp152.3674.1.noarch by ignoring some of its dependencies
```

Does someone has to do some updating before?